server/run-delivery.sh

```diff
@@ -31,7 +31,7 @@ OUTPUT_IMAGE=registry.centos.org/${APPID}/${TO}
 NS="${APPID}-${JOBID}-${DESIRED_TAG}"
 
 _ "Send success mail for user notify tube"
-python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE} ${NOTIFY_EMAIL} ${TEST_TAG} ${NS} ${JOBID} ${LOGS_DIR}
+# python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE} ${NOTIFY_EMAIL} ${TEST_TAG} ${NS} ${JOBID} ${LOGS_DIR}
```

Comment:
It was uncommented and we made it commented. Just keep the file intact? If this is commented out, no email will ever be sent after delivery.